release_notes.md

```diff
@@ -1,3 +1,58 @@
+**********************************
+# Release Notes for OpenMDAO 3.2.0
+
+July 20, 2020
+
+OpenMDAO 3.2.0 introduces significant changes in the way OpenMDAO works.
+The primary change is that the manual creation IndepVarComp outputs are no longer required because OpenMDAO will create them for you in the background.
+This was done in a backwards compatible way so that old models will still run and you can still use IndepVarComps if you want to.
```

Comment:
-> the manual creation of IndepVarComp outputs is no longer....

Reply:
This comment still needs to be addressed.

Reply:
I'll make this change on the release branch.. coming in a minute..